_rules/html-has-lang-b5c3f8.md

```diff
@@ -164,7 +164,7 @@ The `lang` and `xml:lang` attribute specified is empty ("").
 
 #### Inapplicable Example 1
 
-The rule does not apply to `svg` element.
+The rule does not apply to `SVG` element.
```

Comment:
The element is `svg`. SVG (no backtick) is the category of elements, same as HTML.